src/current/cockroachcloud/provision-a-cluster-with-terraform.md

```diff
@@ -179,6 +188,78 @@ The `terraform show` command shows detailed information of your cluster resource
+</div>
+
+{{site.data.alerts.callout_info}}
+To migrate between CockroachDB {{ site.data.products.advanced }} and CockroachDB {{ site.data.products.standard }}, you must create and configure a new cluster, back up the existing cluster's data, and restore the backup to the new cluster. Migration in place is not supported. Refer to [Customer-owned backups]({% link cockroachcloud/backup-and-restore-overview.md %}#customer-owned-backups).
+{{site.data.alerts.end}}
+
+<section class="filter-content" markdown="1" data-scope="terraform">
+
+To migrate from CockroachDB {{ site.data.products.basic }} to CockroachDB {{ site.data.products.standard }} in place:
+
+1. Edit the cluster's Terraform template:
+    - Change `plan` to `STANDARD`.
+    - Replace the contents of `serverless {}` (which may be empty) with the provisioned vCPUs for the cluster. This field is required for CockroachDB {{ site.data.products.standard }}. It is not possible to set storage limitations on CockroachDB {{ site.data.products.standard }}.
+      {% include_cached copy-clipboard.html %}
+      ~~~ hcl
+            serverless = {
+          usage_limits = {
+            provisioned_virtual_cpus = 2
+          }
+        }
+      ~~~
+1. Apply the template:
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    terraform apply
+    ~~~
+
+To migrate from CockroachDB {{ site.data.products.standard }} to CockroachDB {{ site.data.products.basic }} in place:
+
+1. Edit the cluster's Terraform template:
+    -  Change `plan` to `BASIC`.
+    - Replace the contents of `serverless {...}` with optional limits for Request Units and Storage. The `provisioned_virtual_cpus` field is not supported on CockroachDB {{ site.data.products.basic }}.
+      {% include_cached copy-clipboard.html %}
+      ~~~ hcl
+        serverless = {
+          usage_limits = {
+            request_unit_limit = 4000
+            storage_mib_limit = 2000
+          }
+        }
+      ~~~
+    - Remove configurations for features that are unsupported on CockroachDB {{ site.data.products.basic }}, such as private connectivity. Otherwise, applying the template will fail.
+1. Apply the template:
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    terraform apply
+    ~~~
+
+</section>
+
+<section class="filter-content" markdown="1" data-scope="api">
+
+To migrate from CockroachDB {{ site.data.products.basic }} to CockroachDB {{ site.data.products.standard }} in place:
+
+1. Edit the API command or application code that was used to created the cluster. In the JSON header, replace the contents of `serverless {}` (which may be empty) with the provisioned vCPUs for the cluster. This field is required for CockroachDB {{ site.data.products.standard }}. It is not possible to set storage limitations on CockroachDB {{ site.data.products.standard }}. Refer to [CockroachDB API: Create a Standard cluster](https://cockroachlabs.com/docs/api/cloud/v1.html#post-/api/v1/clusters).
```

Comment:
```suggestion
1. Edit the API command or application code that was used to create the cluster. In the JSON header, replace the contents of `serverless {}` (which may be empty) with the provisioned vCPUs for the cluster. This field is required for CockroachDB {{ site.data.products.standard }}. It is not possible to set storage limitations on CockroachDB {{ site.data.products.standard }}. Refer to [CockroachDB API: Create a Standard cluster](https://cockroachlabs.com/docs/api/cloud/v1.html#post-/api/v1/clusters).
```

Also I like the "H1: H2" link format. Haven't thought to use that in our docs before.

Reply:
We have done this a few times when we are linking to a specific section of a long page, just to make the link text a little more accessible. I applied your suggestion elsewhere.